src/main.js

```diff
@@ -342,6 +343,17 @@ app.on('ready', () => {
     console.log('The application has crashed.');
   });
 
+  // Add Alt+Cmd+(Right|Left) as alternative to switch between servers
+  if (process.platform === 'darwin') {
+    globalShortcut.register('Alt+Cmd+Right', () => {
+      mainWindow.webContents.send('select-next-tab');
+    });
+
+    globalShortcut.register('Alt+Cmd+Left', () => {
```

Comment:
Do not use globalShortcut. You can use a  MenuItem with `visible: false` parameter.

Reply:
Unfortunately this solution isn't working on Mac, and if it's still working for Windows it's a bug, see: https://github.com/electron/electron/issues/7737.

I've updated the PR with the `before-input-event` implementation which is mentioned in that thread.